keyboard/src/modules/Keyboard.rb

```diff
@@ -1522,6 +1518,23 @@ def enable_autorepeat
     SCR.Execute(path(".target.bash"), cmd)
   end
 
+  # Keyboards map
+  #
+  # The map can be read from two different files:
+  #
+  # * `keyboard_raw_PRODUCT.ycp` where PRODUCT is the downcased product's
+  #   short name. For example, `keyboard_raw_sles.ycp`.
+  # * `keyboard_raw.ycp` as a fallback.
+  #
+  # @return [Hash] Keyboards map
+  def all_keyboards
+    content = SCR.Read(path(".target.yast2"), "keyboard_raw_#{OSRelease.id}.ycp")
```

Comment:
1) I miss explicit OSRelease import in the file
2) I've tried OSRelease.id in irb and it stated "undefined method `id'"